browser/resources/settings/brave_overrides/autofill_page.ts

```diff
@@ -7,12 +7,26 @@ import {html, RegisterPolymerTemplateModifications, RegisterPolymerComponentRepl
 import {BraveSettingsAutofillPageElement} from '../brave_autofill_page/brave_autofill_page.js'
 import {loadTimeData} from '../i18n_setup.js'
 
+import '../email_aliases_page/email_aliases_page.js'
+
 RegisterPolymerComponentReplacement(
   'settings-autofill-page', BraveSettingsAutofillPageElement
 )
 
+
 RegisterPolymerTemplateModifications({
   'settings-autofill-page': (templateContent) => {
+    const isEmailAliasesFeatureEnabled = loadTimeData.getBoolean('isEmailAliasesFeatureEnabled')
+    if (isEmailAliasesFeatureEnabled) {
+      const parentManagerButton = templateContent.getElementById('paymentManagerButton')
```

Comment:
probably worth an 

```ts
if (!parentManagerButton) {
    console.error('[overrides]: Couldnt find ...')
}
```

to help track down breakages

Reply:
fixed